go/host/node/node.go

```diff
@@ -152,6 +158,12 @@ func NewHost(
 				Service:   clientapi.NewTestAPI(node),
 				Public:    true,
 			},
+			{
+				Namespace: "eth",
+				Version:   apiVersion1,
+				Service:   filters.NewPublicFilterAPI(events.NewBackend(node.logsCh), false, 5*time.Minute),
```

Comment:
We reuse Geth's `PublicFilterAPI` to handle log subscriptions, passing in our own custom backend.